pkgs/development/libraries/libshumate/default.nix

```
@@ -32,6 +32,9 @@ stdenv.mkDerivation rec {
     sha256 = "+h0dKLECtvfsxwD5aRTIgiNI9jG/tortUJYFiYMe60g=";
   };
 
+  # build pkg-config is required to find native gi-docgen when cross compiling
+  depsBuildBuild = [ pkg-config ];
```

Maybe the following for more uniform style and to avoid dissociating the comment when more are added.

```suggestion
  depsBuildBuild = [
    # required to find native gi-docgen when cross compiling
    pkg-config
  ];
```